goss-testing/automated/run-ncn-tests.sh

```diff
@@ -44,6 +44,20 @@ function is_pit_node {
     [[ -f /etc/pit-release ]]
     return $?
 }
+ 
+function is_vshasta_node {
+    # This is the best check for an image specifically booted to vshasta
+    if [[ -f /etc/google_system ]]; then
+      return $?
+    # metal images can still be booted on GCP, so check if there are any disks vendored by Google
+    elif lsblk --noheadings -o vendor | grep Google >/dev/null; then
+      return $?
```

Comment:
Should be the same, but seems more straightforward to not use the variable when we always know it is 0?

```suggestion
      return 0
```

Reply:
This was marked resolved but it looks to me like it is not? Or is github messing with me again? Sometimes it likes to show me stale PR data, I think.

Reply:
Oh yeah, I co-authored the commit with you.  https://github.com/Cray-HPE/csm-testing/pull/414/commits/be2345b6a3bea4ce0080b75151b5e46be7d259e0.

The same thing happens to me, but I took all your suggestions if you check the commits tab.

Reply:
I don't see the above suggestion in the commit tag.

Reply:
https://github.com/Cray-HPE/csm-testing/pull/414/commits/afc2c42471a264a846a023b26ed51e9fddb9954e